apps/ledger-live-desktop/src/newArch/features/WalletSync/hooks/useWatchWalletSync.ts

```diff
@@ -118,6 +126,16 @@ export function useWatchWalletSync(): WalletSyncUserState {
     [visualPending, walletSyncError, onUserRefresh],
   );
 
+  const watchConfig = useMemo(
+    () => ({
+      notificationsEnabled: featureWalletSync?.params?.watchConfig?.notificationsEnabled,
+      pollingInterval: featureWalletSync?.params?.watchConfig?.pollingInterval,
+      initialTimeout: featureWalletSync?.params?.watchConfig?.initialTimeout,
+      userIntentDebounce: featureWalletSync?.params?.watchConfig?.userIntentDebounce,
+    }),
+    [featureWalletSync?.params?.watchConfig],
+  );
```

Comment:
remain to be resolved, right?

Reply:
My bad, I did it for mobile only. Just did it for desktop